dockers/docker-snmp-sv2/snmpd.conf.j2

```diff
@@ -118,6 +118,7 @@ load   12 10 5
 #
                                            #  Run as an AgentX master agent
 master          agentx
+agentxsocket    tcp:localhost:3161
```

Comment:
>3161 [](start = 30, length = 4)

How did you choose this port? I see it also in snmp.conf. Suggest prevent magic numbers, or add comment for reference each other so we will not forget if we want to modify it.

Reply:
standard port is 705, chose 3000+ snmp port to not run into issues for daemons that re not launched as root.
We can make this a configurable option in rules/config?

Reply:
rules/config is good!

Reply:
actually talked to fast: this would require a nested template (dockers/docker-snmp-sv2/snmpd.conf.j2.j2 ) which would add complexity, the best thing I believe is to add it to our plan to integrate snmp config in config_db.json as per https://github.com/Azure/SONiC/pull/231
we should start working on this in Q3 ( within a month ). what do you think?

Reply:
OK to me. Then please check the alternative recommendation
> add comment for reference each other so we will not forget if we want to modify it.

Reply:
agreed

Reply:
added references to both files